apps/src/applab/designMode.js

```diff
@@ -122,12 +122,6 @@ designMode.attachElement = function (element) {
 };
 
 designMode.editElementProperties = function (element) {
-  var designPropertiesElement = document.getElementById('design-properties');
-  if (!designPropertiesElement) {
-    // design-properties won't exist when !user.isAdmin
```

Comment:
This comment wasn't true.  I verified that `#design-properties` is created any time design mode renders itself, no conditions are checked.  Besides, `designPropertiesElement` was unused in this function, except for the early-out, so I'm not sure what it was guarding against.